docs/DeveloperGuide.md

```diff
@@ -283,32 +283,65 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 ### Use cases
 
-(For all use cases below, the **System** is the `AddressBook` and the **Actor** is the `user`, unless specified otherwise)
+(For all use cases below, the **System** is the `Staff'd` and the **Actor** is the `user`, unless specified otherwise)
 
-**Use case: Delete a person**
+**Use case: UC01 - Edit staff details**
 
 **MSS**
 
-1.  User requests to list persons
-2.  AddressBook shows a list of persons
-3.  User requests to delete a specific person in the list
-4.  AddressBook deletes the person
+1.  User chooses to edit the staff details.
+2.  User inputs relevant details.
+3.  Staff’d requests for confirmation.
+4.  User confirms.
+5. Staff’d updates the new staff details.
 
-    Use case ends.
+   Use case ends.
 
 **Extensions**
 
-* 2a. The list is empty.
+* 3a. Staff'd detects an error in the entered data.
 
-  Use case ends.
+    * 3a1. Staff'd displays an error message.
+    * 3a2. User enters new data.
+    * 3a2. Steps 3a1-3a2 are repeated until the data entered are correct.
 
-* 3a. The given index is invalid.
+      Use case resumes at step 4.
 
-    * 3a1. AddressBook shows an error message.
 
-      Use case resumes at step 2.
+* *a At any time, User chooses to cancel the edit.
```

Comment:
How do we cancel the edit when the edit command states the edit? I do not think this extension is feasible. Unless you want to change how edit works?